cpp/src/arrow/ipc/json-test.cc

```diff
@@ -57,6 +57,11 @@ void TestSchemaRoundTrip(const Schema& schema) {
   writer.EndObject();
 
   std::string json_schema = sb.GetString();
+  // RapidJSON can use SIMD operations that read past string
+  // but make ASAN complain, pad to make sure there is enough
+  // memory to keep ASAN happy (and accomodate if rapid json upgrades
+  // to SIMD instructions with 512 byte width).
```

Comment:
Hmm, there are 512-bit SIMD instructions around (AVX512), but not 512-byte I think.

Reply:
Good catch :), also thanks for looking more closely at rapid JSON I've changed the callpoints over to use the safe method.